tests/functional/test_adcm_upgrade.py

```diff
@@ -118,3 +135,21 @@ def test_pass_in_config_encryption_after_upgrade(
 
     _check_encryption(cluster)
     _check_encryption(service)
+
+
+@pytest.mark.parametrize("adcm_is_upgradable", [True], indirect=True)
+@pytest.mark.parametrize("image", [["hub.arenadata.io/adcm/adcm", "2021.06.17.06"]], ids=repr, indirect=True)
```

Comment:
These images will be obtained dynamically?

Reply:
Nope. In this test we use exact 1 image (last release with old DSL)